downstream/modules/platform/con-gw-authenticator-map-triggers.adoc

```diff
@@ -17,14 +19,18 @@ Group:: The map is true or false  based on a user having, not having or having m
 If you are only keying off one group it doesn’t matter if you select *"and"* or *"or"*.
 ====
 +
-* *Groups:* This is a list of one or more groups coming from the authentication system that the user must be a member of. See the *Operation* field to determine the behavior of the trigger if more than one group is specified in the trigger.
+* *Groups:* This is a list of one or more groups coming from the authentication system that the user must be a member of. The first time you create a *Groups* entry, you must manually enter the values. Once entered, that selection will be available from the drop-down list. 
```

Comment:
Minor nits/suggestions

```suggestion
* *Groups:* This is a list of one or more groups coming from the authentication system that the user must be a member of. The first time you create a *Groups* entry, you must manually enter the values. After you enter the values, you can select them from the *Groups* list. 
```
Adhering to the "drop-down list" rule in IBM style guide on [drop-down list](https://www.ibm.com/docs/en/ibm-style?topic=elements-ui#drop_down_list): Refer to a drop-down list by its label, followed by list.